modules/book.js

```diff
@@ -0,0 +1,55 @@
+const titleAuthorArray = [];
+class TitleAuthor {
+  remove(str) {                        // eslint-disable-line
+    const stored = JSON.parse(localStorage.getItem('students'));
+    stored.splice(str, 1);
+    localStorage.setItem('students', JSON.stringify(stored));
+  }
+
+    submit() {            // eslint-disable-line
+
+    const title = document.getElementById('title').value;
+    const author = document.getElementById('author').value;
+        const title_author = {               // eslint-disable-line
+
+      Title: title,
+      Author: author,
+    };
+
+    const storedTitleAuthor = JSON.parse(localStorage.getItem('students'));
+    if (!storedTitleAuthor) {
+      titleAuthorArray.push(title_author);
+      localStorage.setItem('students', JSON.stringify(titleAuthorArray));
+    } else {
+      storedTitleAuthor.push(title_author);
+      localStorage.setItem('students', JSON.stringify(storedTitleAuthor));
+    }
+
+        location.reload(); // eslint-disable-line
```

Comment:
It works, but in order to keep your code professional it's highly recommended to follow the eslint rules, for example in this case you can fix the linter error by adding `window` to the beginning of the `location` like `window.location.reload();` I'd suggest to google for others as well and avoid skipping the linter checks.